### DISCUSSION TITLE
StrumPract would need an user guide

### DISCUSSION THREAD

**Discussion by rchastain** [fredvs/strumpract#16] [Announcements]
Hello Fred (and others)! I downloaded and compiled without problem the latest release of StrumPract. It looks very beautiful, but I have no idea of what I could do with it. When you have time, could you make a small user guide? Or maybe a video demonstration?

Best regards.

MC Roland

**Comment by fredvs** [comment]
Hello Roland.

Thanks to try Strumpract.

Strumpract is the Swiss pocketknife of the musicians.

You may begin with the DJ feature.

On the main menu, click on Layout > DJ Console.

You will have 2 players with wave forms and freqVU, a mixing table, a list of songs.

First, fill the list of songs, clicking on the [...] button of the Audio Files window.
Choose a directory whith songs and click OK.

Now you may do your mixes, clicking on the [>] orange button in the mixing table.
It will automatically do the mixes at end of sound.
You may set the duration of the mix with the MT button.

If you want you may stretch the song (changing the tempo without changing the pitch).
Ideal if you want to discover some note or chord of a song, or play with it but with a tempo slower.

You may try different layouts, play with the docking feature, save your own layout, etc.

I dont tell you all, otherwise there are no more surprises...

**Comment by rchastain** [comment]
Thank you Fred. I will try what you say and come back.

**Comment by rchastain** [comment]
By the way Fred, you could add the topic _mse_ or _mseide_ or _msegui_ to your project page. And also something in the README, so that people know that it is an MSEide project?

**Comment by fredvs** [comment]
Of couuuuurse!

And it is a shame that I did not add this before.
I will correct this tonight.

Thanks to note that unforgivable omission.

[EDIT] Done.

Fre;D
